migration/migrator/migrations/master/20250121144500_courses_users_indexes.py

```diff
@@ -0,0 +1,26 @@
+"""Migration for the Submitty master database."""
+
+
+def up(config, database):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    """
+    database.execute("CREATE INDEX IF NOT EXISTS courses_users_user_id_idx ON courses_users (user_id);")
+    database.execute("CREATE INDEX IF NOT EXISTS courses_users_user_group ON courses_users (user_id, user_group);")
```

Comment:
Given that this is a pretty small table overall, I think having a dedicated index on `(user_id)` is probably redundant.  The database can use the `(user_id, user_group)` index to achieve the same result with basically no difference.

Also, what do you think about renaming the second index to `courses_users_user_id_user_group_idx`?